Makefile

```
@@ -71,8 +85,17 @@ gen_stream_tb:	$(STREAM_GEN_OUT_SCALA_FILE) $(STREAM_GEN_OUT_TOP_FILE) $(STREAM_
 	--tpl_path="${STREAM_GEN_TPL_TB_FILE}" \
 	--out_path="${STREAM_GEN_OUT_TB_FILE}"
 
+#-----------------------------
+# Generate Streamer-TCDM Wrapper Testbench
+#-----------------------------
+gen_stream_tcdm_tb:	$(STREAM_GEN_OUT_SCALA_FILE) $(STREAM_GEN_OUT_TOP_FILE) $(STREAM_GEN_OUT_RTL_FILE)
```

Should this recipe not have the name of the generated file?
```suggestion
$(STREAM_GEN_OUT_TB_FILE): $(STREAM_GEN_OUT_SCALA_FILE) $(STREAM_GEN_OUT_TOP_FILE) $(STREAM_GEN_OUT_RTL_FILE)
```

---

Good point haha why did I not do that last time aksdjalksdjaslkdjaslkdjl what was I thinking 🍩